docs/api/api_docs.yaml

```diff
@@ -64,124 +99,616 @@ paths:
     get:
       responses:
         '200':
-          description: OK
+          $ref: '#/components/responses/all-docs'
+        '400':
+          $ref: '#/components/responses/request-problem'
+        '404':
+          $ref: '#/components/responses/Not-found'
       tags:
         - Admin
         - Public
+      parameters:
+        - $ref: '#/components/parameters/include_docs'
+        - $ref: '#/components/parameters/Include-channels'
+        - $ref: '#/components/parameters/include-access'
+        - $ref: '#/components/parameters/include-revs'
+        - $ref: '#/components/parameters/include-seqs'
+        - $ref: '#/components/parameters/keys'
+        - $ref: '#/components/parameters/startkey'
+        - $ref: '#/components/parameters/endkey'
+        - $ref: '#/components/parameters/limit'
+      summary: Get all the documents in the database using a built-in view
+      description: Get a built-in view of all the documents in the database.
```

Comment:
The mechanism for retrieval doesn't matter (and is not always a view). Just say that the endpoint returns all documents for the given parameters.